fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/literal/DateLiteral.java

```diff
@@ -204,4 +214,13 @@ public DateLiteral plusYears(int years) {
         LocalDateTime dateTime = DateUtils.getTime(DATE_FORMATTER, getStringValue()).plusYears(years);
         return new DateLiteral(dateTime.getYear(), dateTime.getMonthValue(), dateTime.getDayOfMonth());
     }
+
+    public LocalDateTime toJavaDateType() {
+        return LocalDateTime.of(((int) getYear()), ((int) getMonth()), ((int) getDay()), 0, 0, 0);
+    }
+
+    public static DateLiteral fromJavaDateType(LocalDateTime dateTime) {
+        return isDateOutOfRange(dateTime) ? null
```

Comment:
do not return null, return NullLiteral or throw exception

Reply:
it will throw exception